posts/models.py

```diff
@@ -9,5 +9,8 @@ class Post(models.Model):
 	author = models.ForeignKey('users.User', on_delete= models.CASCADE)
 	stream = models.ForeignKey('streams.Stream', default=None, on_delete=models.CASCADE, db_column="title")
 
+	class Meta:
+		ordering = ['-pub_date']
```

Comment:
Wouldn't it be better to order by last modified ( descending ) ?

Reply:
I asked Divyanshu about this, he said it would be better to use publishing date